ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1079,3 +1079,18 @@ literals:
 - zone_air_no2_probability_percentage_sensor
 - zone_air_no2_ozone_concentration_sensor
 
+## Geothermal loop fields
+- geothermal_return_water_temperature_sensor
+- geothermal_supply_water_temperature_sensor
+- geothermal_thermal_energy_accumulator
+- geothermal_thermal_power_sensor
+- geothermal_water_settling_temperature_sensor
+- geothermal_water_flowrate_sensor
+
+## PV meter fields
+- photovoltaic_cell_charge_energy_accumulator
```

Comment:
is a use for these fields forthcoming? i would not want to commit additional fields without downstream utilization in types.